transformer_engine/pytorch/float8_tensor.py

```diff
@@ -307,8 +334,9 @@ def __new__(
         ), f"Unsupported fp8_dtype {fp8_dtype}."
         self._fp8_dtype: tex.DType = fp8_dtype
 
-        # Cached transpose
+        # Transposed version of `_data`.
         self._transpose: Optional[Float8Tensor] = None
+        self._transpose_invalid: bool = True
```

Comment:
It would be easier to fill the cache externally, e.g. in a fused `DgeluDbias` op, if the cache were valid by default.
```suggestion
        self._transpose_invalid: bool = False
```

Reply:
The default value here depends on how this attr is interpreted. Since we don't have a valid transpose stored during initialization, to me it makes sense for this to be `True` to begin with.

Reply:
That's reasonable, although it adds one more thing to handle if the transpose is set externally.